src/openzaak/components/catalogi/api/filters.py

```diff
@@ -86,6 +86,11 @@ class RolTypeFilter(FilterSet):
         field_name="zaaktype__identificatie",
         help_text=get_help_text("catalogi.ZaakType", "identificatie"),
     )
+    omschrijving = filters.CharFilter(
+        field_name="omschrijving",
+        lookup_expr="icontains",
+        help_text=mark_experimental("Filter op omschrijving (hoofdletterongevoelig)."),
```

Comment:
I read over the fact that we use icontains, so let's mention that you can also search for parts of the `omschrijving`
```suggestion
        help_text=mark_experimental("Filter op (een deel van de) omschrijving (hoofdletterongevoelig)."),
```